xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/api/XWiki.java

```diff
@@ -1283,7 +1283,7 @@ public void flushCache()
     /**
      * Privileged API to create a new user from the request This API is used by RegisterNewUser wiki page
      *
-     * @return true for success/false for failure
+     * @return integer status code
```

Comment:
Also you should indicate what the integer means or point to a place that describes it.